Tests/MapboxMapsTests/Annotations/ViewAnnotationManagerTests.swift

```diff
@@ -248,6 +272,70 @@ final class ViewAnnotationManagerTests: XCTestCase {
         XCTAssertTrue(annotationViewC.isHidden)
     }
 
+    func testViewAnnotationUpdateObserverNotifiedAboutUpdatedFrames() {
+        let annotationView = addTestAnnotationView()
+        let id = mapboxMap.addViewAnnotationStub.invocations.last!.parameters.id
```

Comment:
Avoid crashing tests:

```suggestion
        let id = try XCTUnwrap(mapboxMap.addViewAnnotationStub.invocations.last?.parameters.id)
```